### ISSUE TITLE
NativeModules.DatabaseBridge is not defined

### ISSUE THREAD

**Issue by starfreck** [Nozbe/WatermelonDB#1580]
I started facing the same issue which is mentioned [here](https://github.com/Nozbe/WatermelonDB/issues/1408) with Expo 48.0.10

"dependencies": {
"@morrowdigital/watermelondb-expo-plugin": "^2.1.0",
"@nozbe/watermelondb": "^0.25.5",
"@pchmn/expo-material3-theme": "^1.0.1",
"@react-native-community/masked-view": "^0.1.11",
"@react-navigation/native": "^6.1.6",
"@react-navigation/stack": "^6.3.16",
"axios": "^1.3.5",
"dotenv": "^16.0.3",
"expo": "^48.0.10",
"expo-constants": "~14.2.1",
"expo-secure-store": "~12.1.1",
"expo-splash-screen": "~0.18.1",
"expo-sqlite": "~11.1.1",
"expo-status-bar": "~1.4.4",
"react": "18.2.0",
"react-native": "0.71.6",
"react-native-gesture-handler": "~2.9.0",
"react-native-keyboard-aware-scroll-view": "^0.9.5",
"react-native-safe-area-context": "^4.5.0",
"react-native-screens": "^3.20.0",
"react-native-svg": "13.4.0",
"react-navigation": "^4.4.4",
},

Does anyone know how to fix it?

**Comment by GeorgeHop** [comment]
3 weeks and no response from maintainers. Looks like this lib. is dead.

**Comment by GeorgeHop** [comment]
With my expo setup it's don't event start https://github.com/Nozbe/WatermelonDB/issues/1592 
You solved your issue and how? And which flow for expo do you use? ejected or managed?

**Comment by eduardolsb** [comment]
Expo is dead!!!

**Comment by GeorgeHop** [comment]
Looks like this lib. is dead)

**Comment by skam22** [comment]
@starfreck this should solve your issue:

https://github.com/Nozbe/WatermelonDB/issues/1592#issuecomment-1551882772

**Comment by starfreck** [comment]
> Looks like this lib. is dead)

I was able to run the project with [expo-dev-client](https://docs.expo.dev/develop/development-builds/introduction/#what-is-expo-dev-client) but it won't run in Expo Go.

https://github.com/starfreck/React-Native-Expo-WatermelonDB-Demo

@GeorgeHop I have decided not to use this lib in my project. It seems like the development is going at a very slow pace. Also, it lacks good examples in docs for the new features. Most examples that you will find have some parts which are deprecated in the new version.  As an alternative, I have decided to use the below two libs and which made my life very easy. Hope it helps others in their projects.

Thank you @skam22 for the response.

Cheers 🤞🏼

https://www.npmjs.com/package/expo-sqlite
https://www.npmjs.com/package/expo-sqlite-orm
